lib/mix/tasks/phoenix.new.ex

```diff
@@ -42,30 +51,73 @@ defmodule Mix.Tasks.Phoenix.New do
         mod = opts[:module] || Naming.camelize(app)
         check_mod_name!(mod)
 
-        run(app, mod, path, opts[:dev])
+        run(app, mod, path, opts[:dev], opts[:skip_brunch], opts[:skip_npm])
```

Comment:
At this point we should just pass all `opts`? Don't worry though, I can refactor it when working on the model generator.